tests/test_image/impad.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+
+
+def impad(img, shape, pad_val=0):
```

Comment:
It can be removed if the new version is confirmed to be the same with the old one.

Reply:
It has been removed in the latest version.